lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -658,6 +716,7 @@ WasmBinaryReader::LEB128(UINT &length, bool sgn)
     UINT result = 0;
     UINT shamt = 0;
     byte b;
+    length = 0;
```

Comment:
From what I saw in the way we used LEB128, we assumed that the length passed would always have the number of bytes read for this LEB128.
That wasn't the case and introduced bugs.
It could have been a valid use, but since at least 2 dev made that mistake, I think it better to always initialize the length to 0 when reading an LEB128